<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMapIconSizesSetup</class>
 <widget class="QDialog" name="IMapIconSizesSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Set Map Icon Sizes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QSlider" name="sliderWpt">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>48</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="unitPoi">
       <property name="text">
        <string>px</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="titleWpt">
       <property name="text">
        <string>Waypoint Icons</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="valuePoi">
       <property name="text">
        <string notr="true">111</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="sizeWpt">
       <property name="text">
        <string>Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="unitWpt">
       <property name="text">
        <string>px</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="titlePoi">
       <property name="text">
        <string>POI Icons</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="sliderPoi">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>48</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="sizePoi">
       <property name="text">
        <string>Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="valueWpt">
       <property name="text">
        <string notr="true">111</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3" rowspan="2">
      <widget class="QLabel" name="labelPoi">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QLabel" name="labelWpt">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IMapIconSizesSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IMapIconSizesSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
